cmd/multiFaExtract/multiFaExtract_test.go

```diff
@@ -12,25 +12,69 @@ var MultiFaExtractTests = []struct {
 	outputFile   string
 	expectedFile string
 	removeGaps   bool
+	bed          string
 	start        int
 	end          int
+	out1         string //these six fields are for the three output and expected files made by the bed option usage of this cmd.
+	out2         string
+	out3         string
+	expected1    string
+	expected2    string
+	expected3    string
 }{
-	{"testdata/testInput.fa", "testdata/testOut.fa", "testdata/testOut.10to200.fa", false, 10, 200},
-	{"testdata/testInput.fa", "testdata/testOut.fa", "testdata/testOut.10to200.RemoveGaps.fa", true, 10, 200},
+	{"testdata/testInput.fa", "testdata/testOut.fa", "testdata/testOut.10to200.fa", false, "", 10, 200, "", "", "", "", "", ""},
+	{"testdata/testInput.fa", "testdata/testOut.fa", "testdata/testOut.10to200.RemoveGaps.fa", true, "", 10, 200, "", "", "", "", "", ""},
+	{"testdata/testInput.fa", "", "", false, "testdata/test.bed", -1, -1, "chr1.20.30.fa", "chr1.30.50.fa", "chr1.60.200.fa", "testdata/chr1.20.30.expected.fa", "testdata/chr1.30.50.expected.fa", "testdata/chr1.60.200.expected.fa"},                  //~/go/bin/multiFaExtract -bed testdata/test.bed testdata/testInput.fa
+	{"testdata/testInput.fa", "", "", true, "testdata/test.bed", -1, -1, "chr1.20.30.fa", "chr1.30.50.fa", "chr1.60.200.fa", "testdata/chr1.20.30.expected.noGap.fa", "testdata/chr1.30.50.expected.noGap.fa", "testdata/chr1.60.200.expected.noGap.fa"}, //~/go/bin/multiFaExtract -bed testdata/test.bed -removeGaps testdata/testInput.fa
 }
 
 func TestMultiFaExtract(t *testing.T) {
 	var err error
+	var s Settings
 	for _, v := range MultiFaExtractTests {
-		multiFaExtract(v.inputFile, v.outputFile, v.start, v.end, v.removeGaps)
-		records := fasta.Read(v.outputFile)
-		expected := fasta.Read(v.expectedFile)
-		if !fasta.AllAreEqual(records, expected) {
-			t.Errorf("Error in multiFaExtract.")
+		s = Settings{
+			InFile:     v.inputFile,
+			OutFile:    v.outputFile,
+			Start:      v.start,
+			End:        v.end,
+			Bed:        v.bed,
+			RemoveGaps: v.removeGaps,
 		}
-		err = os.Remove(v.outputFile)
-		if err != nil {
-			common.ExitIfError(err)
```

Comment:
What was the logic in switching from the common package error function to the exception package error function?

Reply:
the exception error contains the check for if err != nil inside of the function. Should function the same but is more concise